api/app/schemas/observations.py

```diff
@@ -19,6 +19,8 @@ class WeatherReading(BaseModel):
     ffmc: float = None
     isi: float = None
     fwi: float = None
+    observation_valid_ind: bool = None
```

Comment:
I guess we're just using the "ind" thing from wfwx? I wonder if we can't just have observation_valid - I mean - obviously it's an indicator? What do you think? Or do we try to match the wfwx convention?

Reply:
I'm good either way, observation_valid is a bit more readable.